code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -61,6 +61,10 @@
 		if(H.dna.features["tail_human"] == "Cat")
 			var/obj/item/organ/tail/cat/tail = new
 			tail.Insert(H, drop_if_replaced = FALSE)
+		else if(H.dna.features["tail_human"] == "Wolf")
+			var/obj/item/organ/tail/cat/wolf/tail = new
+			tail.Insert(H, special = TRUE, drop_if_replaced = FALSE)
+			mutant_organs += /obj/item/organ/tail/cat/wolf
```

Comment:
Explain why this is necessary?

Reply:
...Oh, I see what's up here.

Please restructure this to not result in an incorrect list entry.